gtk/src/dialogs/mod.rs

```diff
@@ -31,7 +32,8 @@ impl FirmwareUpdateDialog {
         let mut header = ["Firmware version ", version, " is available."].concat();
 
         if has_battery {
-            header.push_str(" <b>Plug into power</b> before you begin.");
+            header.push_str(" Connect your computer to power. <b>USB Type-C</b> charging is not supported for firmware updates.");
+	        header = (&*fill(&header, 75)).to_string();
```

Comment:
This is exactly one space off now

Reply:
It should look correct after commit:
https://github.com/pop-os/firmware-manager/pull/77/commits/e8494d07d75b9a01e42ac6295587888c092185fc